spec/unit/trackable_spec.rb

```diff
@@ -621,6 +621,19 @@ def self.name
     end
   end
 
+  describe '#track_history_for_action' do
+    before(:all) { MyModel.track_history }
+    let(:m) { MyModel.create!(foo: 'bar') }
+
+    it 'should yield block' do
+      expect { |b| m.send(:track_history_for_action, :update, &b) }.to yield_control
+    end
+
+    it 'should not raise error when no block' do
+      expect { m.send(:track_history_for_action, :update) }.not_to raise_error
+    end
+  end
+
```

Comment:
I think you should rewrite the tests at a higher level, so not calling `track_history_for_action` via a `send`, but `track_history`, because that is an internal function and we want to make sure the code works at the public  interface level.